docs/source/examples/text.rst

```diff
@@ -0,0 +1,12 @@
+Text
+=================================
+
+.. manim:: Text1
+    :quality: medium
+    :save_last_frame:
+
+    class Text1(Scene):
+        def construct(self):
+            t = TextMobject("Hello World")
+            self.add(t)
```

Comment:
Based on its simplicity, this is an example I would expect rather in the reference manual in the docstring of the `Tex` class.

But I do understand that you are mainly trying to establish some initial files, so I think its fine for now. `TextMobject` should be changed to `Tex`, though.

Reply:
Maybe it should be even changed to "Text()". 
But let's keep this for later